<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>quantEstoque</class>
 <widget class="QMainWindow" name="quantEstoque">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="inserirProdutoNaTabela">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>371</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Inserir</string>
    </property>
   </widget>
   <widget class="QPushButton" name="compararPorNome">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>230</y>
      <width>161</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Ordenar por nome</string>
    </property>
   </widget>
   <widget class="QPushButton" name="compararPorQuantidade">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>230</y>
      <width>181</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Ordenar por quantidade</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="tabelaDados">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>371</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Quantidade no estoque</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>270</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Maior quantidade</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>290</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>270</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Menor quantidade</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_3">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>290</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>370</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="InserirProduto">
         <property name="text">
          <string>Nome do produto</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Qtde Estoque</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTextBrowser" name="entradaNome"/>
       </item>
       <item>
        <widget class="QTextBrowser" name="entradaQuant"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGest_o">
    <property name="title">
     <string>Gestão</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuEstat_stica">
    <property name="title">
     <string>Estatística</string>
    </property>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuGest_o"/>
   <addaction name="menuEstat_stica"/>
  </widget>
  <action name="actionquantidade_de_estoque">
   <property name="text">
    <string>Quantidade de estoque</string>
   </property>
  </action>
  <action name="actionMaior_quantidade">
   <property name="text">
    <string>Maior quantidade</string>
   </property>
  </action>
  <action name="actionMenor_quantidade">
   <property name="text">
    <string>Menor quantidade</string>
   </property>
  </action>
  <action name="actionEstoque">
   <property name="text">
    <string>Estoque</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
